main.cpp

```diff
@@ -250,7 +487,10 @@ struct ping_queue_t
 
 		*out = m_queue.front();
 		m_queue.pop_front();
-		m_ips.erase(out->ep);
+		if (out->ep.protocol() == udp::v4())
```

Comment:
actually, I'm interested in making it possible to run the bootstrapper on small machines, and blowing up the entry for IPv4 peers may make that problematic